ceph-salt-formula/salt/_states/ceph_orch.py

```diff
@@ -25,14 +24,14 @@ def wait_for_admin_host(name, timeout=1800):
             if provisioned:
                 configured_ret = __salt__['cmd.run_all']("ssh -o StrictHostKeyChecking=no "
                                                         "-i /tmp/ceph-salt-ssh-id_rsa root@{} "
-                                                        "'salt-call ceph_orch.configured "
-                                                        "--out=json --out-indent=-1'".format(
+                                                        "'if [[ -f /etc/ceph/ceph.conf "
+                                                        "&& -f /etc/ceph/ceph.client.admin.keyring ]]; "
+                                                        "then timeout 15 ceph orch status; "
```

Comment:
Could we double the timeout to 30 seconds?

Reply:
Done